gstools/krige/krigesum.pyx

```diff
@@ -37,3 +37,28 @@ def krigesum(
             field[k] += cond[i] * krig_fac
 
     return np.asarray(field), np.asarray(error)
+
+
+def krigesumfast(
```

Comment:
I think a short comment on what the difference to `krigesum` is should be written here.
Have you measured the difference? - Maybe it's more `krigesumwithlessmemoryconsumption`? ;-)

Reply:
I am also not happy. Got a better name/idea?

Reply:
Hmm, what about something in the direction of `calc_krige_and_error_field` and `calc_krige_field`?

I was just thinking, what about combining both functions with a bool `error` or `error_field` or `calc_error_field` as an argument? - I mean, with predictive branching, it shouldn't be much slower and with `error = False`, the array could be set to `cdef double[:] error = np.zeros(1)` or something for saving memory.
I guess that would have to be profiled to see if it actually is fast.
But that's also something very low on the priority list.

Reply:
Renamed to `calc_field_krige_and_variance` and `calc_field_krige`. Added method `_summate` to `Krige` to care about selection of the correct routine.